app/views/expenses/ExpensesInterruptPageView.scala.html

```diff
@@ -0,0 +1,68 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.Layout
+@import views.html.templates.helpers.ErrorSummary
+@import views.html.templates.helpers.Button
+@import views.html.templates.helpers.Heading
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.templates.helpers.InputText
+@import views.html.templates.helpers.Heading
+
+
+@this(
+        layout: Layout,
+        button: Button,
+        heading: Heading
+)
+
+@(taxYear: Int)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+
+@titleForUse = @{messages("checkYourExpenses.employmentExpenses")}
+@headingForUse = @{titleForUse}
+
+
+ @contentHtml = {
+ <div class="govuk-panel govuk-panel--interruption">
+            <form action=@Some(controllers.expenses.routes.CheckEmploymentExpensesController.show(taxYear).url) class="form">
+                <fieldset class="govuk-fieldset" aria-describedby="start_new_claim">
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl"><h1 class="govuk-heading-xl"><span>@heading(headingForUse, Some(messages("employment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-4")</span>
+                    </h1>
+                    </legend>
+                    <div class="govuk-form-group govuk-!-margin-bottom-9">
+                        <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details1.${if(user.isAgent) "agent" else "individual"}")}</p>
+
+                        <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details2.${if(user.isAgent) "agent" else "individual"}")}</p>
+
+                        <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details3.${if(user.isAgent) "agent" else "individual"}")}</p>
+                    </div>
+                    <button class="govuk-button" data-module="govuk-button">Continue</button>
+                </fieldset>
+            </form>
+ </div>
```

Comment:
Whilst the code we have here is fine - it can be improved by using the helpers that we already have to reduce the lines of necessary code. Also there is a rogue aria-describedby which would be an accessibility error.

I believe this should be the equivalent code using the helpers.
```suggestion
 <div class="govuk-panel govuk-panel--interruption">
        @formHelper(action = controllers.expenses.routes.CheckEmploymentExpensesController.show(taxYear)) {

            @heading(headingForUse, Some(messages("employment.caption", (taxYear - 1).toString, taxYear.toString)), "govuk-!-margin-bottom-5")

            <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details1.${if(user.isAgent) "agent" else "individual"}")}</p>

            <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details2.${if(user.isAgent) "agent" else "individual"}")}</p>

            <p class="govuk-body">@{messages(s"expenses.startEmploymentExpenses.details3.${if(user.isAgent) "agent" else "individual"}")}</p>

            @button(classes = Some("govuk-!-margin-top-7"))
        }
    </div>
```

You'd also need to add form helper to here
```
 @this(
        layout: Layout,
        button: Button,
        heading: Heading,
        formHelper: FormWithCSRF
)
```